{
  "artifact_3371949483": {
    "java_home": "extracted/jdk-25",
    "java_version_output": "openjdk 25-internal 2025-09-16\nOpenJDK Runtime Environment (build 25-internal-iklam-2a5bdd829703bffffc4ff75fe62bb8629bf022a4)\nOpenJDK 64-Bit Server VM (build 25-internal-iklam-2a5bdd829703bffffc4ff75fe62bb8629bf022a4, mixed mode, sharing)",
    "failed_step": "build"
  },
  "status": "failed",
  "history": [
    "0182a9884debde3761d9369d552dc8ef0fcaa4e5.json",
    "10e94797a8def733f14173626bdd89a059f4df08.json",
    "39839d513be64e35e6edeb4d4a2be68b6f170b52.json",
    "52ea7e97c010f2109a776c378dd2b7bb3bad471e.json",
    "cc451e7e03c2b311d0204770c59c37b2e8f8f386.json",
    "d91a6f37303831f4c14400b46feeabcf2fa0358a.json"
  ],
  "datetime": "2025-06-20T17:55:44.172561+00:00",
  "url": "https://github.com/openjdk/jdk/pull/25522",
  "number": 25522,
  "title": "8344165: Trace exceptions with a complete call-stack",
  "head_sha": "2a5bdd829703bffffc4ff75fe62bb8629bf022a4",
  "run_url": "https://github.com/dougxc/openjdk-pr-canary/actions/runs/15784446981"
}